app/templates/tools/includes/list.html

```diff
@@ -46,14 +46,18 @@ <h2 class="heading-medium">Your tools</h2>
           Not deployed
         {% endif %}
       </td>
+        {% for availableTool in availableTools %}
+      <tr>
       <td class="align-right no-wrap">
         {% if not rstudio_is_deploying %}
-          <form action="{{ url_for('tools.deploy', { name: 'rstudio' }) }}" method="post" class="inline-form clearfix">
-            <input type="submit" class="button button-secondary right" value="Deploy" />
+          <form action="{{ url_for('tools.deploy', { name: availableTool.tool }) }}" method="post" class="inline-form clearfix">
+            <input type="submit" class="button button-secondary right" value="Deploy {{ availableTool.label }}" />
```

Comment:
Given the name of the tool in that row is already in the first column, do we need to add it to the button label? It's a genuine question, it may be nice to safe some possibly precious space.